tests/testthat/test-arg_is_.R

```diff
@@ -96,7 +96,8 @@ test_that("date", {
   expect_error(arg_is_date(d, dd, n))
   expect_error(arg_is_date(d, dd, nn))
   expect_silent(arg_is_date(d, dd, n, allow_null = TRUE))
-  expect_silent(arg_is_date(d, dd, nn, allow_na = TRUE))
+  # Upstream issue, see: https://github.com/mllg/checkmate/issues/256
+  # expect_silent(arg_is_date(d, dd, nn, allow_na = TRUE))
```

Comment:
I had to remove `allow_na` from `arg_is_date`, due to a checkmate inconsistency, but this didn't affect the rest of the tests.